.changelog/45217.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:new-resource
+aws_iam_outbound_web_identity_federation
```

Comment:
would it make sense to call this `aws_iam_outbound_web_identity_federation_status` to make it clear that this resource is only about turning it off/on?

Reply:
We usually follow the [API Action](https://docs.aws.amazon.com/IAM/latest/APIReference/API_EnableOutboundWebIdentityFederation.html) to name resources.